recipes/blast/build.sh

```diff
@@ -68,31 +63,32 @@ export AR="${AR} rcs"
     --without-vdb \
     --with-z=$PREFIX \
     --with-bz2=$PREFIX \
-    --with-boost=$PREFIX \
-    --without-gcrypt \
-    --with-nettle=$PREFIX \
     --with-z=$PREFIX \
     --without-krb5 \
     --without-openssl \
-    --with-gnutls=$PREFIX
+    --without-gnutls \
+    --without-gcrypt
 
-projects="algo/blast/ app/ objmgr/ objtools/align_format/ objtools/blast/"
+apps="blastp.exe blastn.exe blastx.exe tblastn.exe tblastx.exe psiblast.exe rpsblast.exe rpstblastn.exe"
+apps="$apps makembindex.exe segmasker.exe dustmasker.exe windowmasker.exe deltablast.exe"
+apps="$apps makeblastdb.exe blastdbcmd.exe blastdb_aliastool.exe convert2blastmask.exe blastdbcheck.exe"
+apps="$apps makeprofiledb.exe blast_formatter.exe"
 cd ReleaseMT
 
 # The "datatool" binary needs the libs at build time, create
 # link from final install path to lib build dir:
 ln -s $SRC_DIR/c++/ReleaseMT/lib $LIB_INSTALL_DIR
 
 cd build
-make -j${CPU_COUNT} -f Makefile.flat all_projects="$projects"
+make -j${CPU_COUNT} -f Makefile.flat $apps
 
 # remove temporary link
 rm $LIB_INSTALL_DIR
 
 mkdir -p $PREFIX/bin $LIB_INSTALL_DIR
-rm $SRC_DIR/c++/ReleaseMT/bin/*_unit_test
```

Comment:
I take it it no longer installs those? That's good.

Reply:
That is indeed the case.